pkg/iac-providers/terraform/v12/load-dir_test.go

```diff
@@ -34,44 +33,44 @@ func TestLoadIacDir(t *testing.T) {
 		dirPath string
 		tfv12   TfV12
 		want    output.AllResourceConfigs
-		wantErr error
+		wantErr bool
```

Comment:
same thing

Reply:
This change was made because we do not want a generic error to be propagated back to the caller functions.